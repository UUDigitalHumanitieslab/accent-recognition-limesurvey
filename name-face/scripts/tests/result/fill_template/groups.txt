<row>
    <gid><![CDATA[1086]]></gid>
    <sid><![CDATA[900000]]></sid>
    <group_name><![CDATA[Intro]]></group_name>
    <group_order><![CDATA[0]]></group_order>
    <description><![CDATA[<p>Welkom</p>
]]></description>
    <language><![CDATA[en]]></language>
    <randomization_group/>
    <grelevance/>
</row>
<row>
    <gid><![CDATA[1002]]></gid>
    <sid><![CDATA[900000]]></sid>
    <group_name><![CDATA[Q1002]]></group_name>
    <group_order><![CDATA[1002]]></group_order>
    <description/>
    <language><![CDATA[en]]></language>
    <randomization_group/>
    <grelevance/>
</row>
<row>
    <gid><![CDATA[1003]]></gid>
    <sid><![CDATA[900000]]></sid>
    <group_name><![CDATA[Q1003]]></group_name>
    <group_order><![CDATA[1003]]></group_order>
    <description/>
    <language><![CDATA[en]]></language>
    <randomization_group/>
    <grelevance/>
</row>
