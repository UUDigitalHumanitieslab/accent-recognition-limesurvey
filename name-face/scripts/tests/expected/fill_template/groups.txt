<row>
    <gid><![CDATA[1000]]></gid>
    <sid><![CDATA[900000]]></sid>
    <group_name><![CDATA[Intro]]></group_name>
    <group_order><![CDATA[0]]></group_order>
    <description><![CDATA[<p>Welkom</p>
]]></description>
    <language><![CDATA[en]]></language>
    <randomization_group/>
    <grelevance/>
</row>
<row>
    <gid><![CDATA[1001]]></gid>
    <sid><![CDATA[900000]]></sid>
    <group_name><![CDATA[Q1001]]></group_name>
    <group_order><![CDATA[1001]]></group_order>
    <description/>
    <language><![CDATA[en]]></language>
    <randomization_group/>
    <grelevance/>
</row>
<row>
    <gid><![CDATA[1002]]></gid>
    <sid><![CDATA[900000]]></sid>
    <group_name><![CDATA[Q1002]]></group_name>
    <group_order><![CDATA[1002]]></group_order>
    <description/>
    <language><![CDATA[en]]></language>
    <randomization_group/>
    <grelevance/>
</row>
